src/custom/utils/index.ts

```diff
@@ -88,3 +88,9 @@ export function getEtherscanLink(chainId: ChainId, data: string, type: BlockExpl
     return getEtherscanUrl(chainId, data, type)
   }
 }
+
+// Shortens OrderID (or any string really) removing initial 2 characters e.g 0x
+// and cutting string at 'chars' length, default = 8
+export function shortenOrderId(orderId: string, chars = 8): string {
+  return orderId.substring(2, chars)
+}
```

Comment:
Is it possible the orderId will ever not have the `0x` prefix?
In case, might be safer to "trim" `0x` from the start only if present

Reply:
Actually. Why don't we trim it when we receive it from the backend? We normalize the orderId when is returned.